frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/pageComponents/PTSelectWard.js

```
@@ -0,0 +1,208 @@
+    if (userType === "employee") {
+      const errorsPresent = !!Object.keys(localFormState.errors).lengtha;
+      if (errorsPresent && !formState.errors?.[config.key]) setError(config.key, { type: "required" });
+      else if (!errorsPresent && formState.errors?.[config.key]) clearErrors(config.key);
+    }
+  }, [localFormState]);
+
+  if (userType === "employee") {
+    return (
+    <React.Fragment>
+        <LabelFieldPair style={{ alignItems: "center", display: "block" }}>
+          <CardLabel className="card-label-smaller" style={{display:"contents",marginBottom:"0px",lineHeight:"inherit"}}>{t("Ward") + " *"}</CardLabel>
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)